.buildkite/commands/release-build-jetpack.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+
+# FIXIT-13.1: Temporary fix until we're on the Xcode 13.1 VM
+echo "--- :rubygems: Fixing Ruby Setup"
+gem install bundler
+
+echo "--- :arrow_down: Installing Release Dependencies"
+brew update # Update homebrew to temporarily fix a bintray issue
+brew install imagemagick
+brew install ghostscript
+
+echo "--- :rubygems: Setting up Gems"
+install_gems
+
+echo "--- :cocoapods: Setting up Pods"
+install_cocoapods
+
+echo "--- :closed_lock_with_key: Installing Secrets"
+bundle exec fastlane run configure_apply
```

Comment:
Have you considered extracting all this boilerplate in a file to `source`?

I think it would still work in Buidkite, but if it didn't, we could update the `command` node from calling this script only to first calling the new shared setup and then running the remaining code (below this comment).

I think duplicating the setup keeps each implementation flexible, but I wonder how much flexibility we need here?

Conversely, here we have two fixes that we'll need to address in the future (The Xcode 13.0 Ruby setup and the `brew update` call) which we'll need to replicate across all commands when the time comes. Granted, it's a simple, mechanical, one-time (🙏 ) operation, so it wouldn't be the end of the world. 😄

Reply:
This is a good question – I considered some DRY for this, but was worried that if these diverge for other reasons we might end up with a mess, hence keeping them separate from the start.

I think once we're able to clean out the `brew` and `install bundler` stuff, we might be able to do your idea of a `prepare_dependencies.sh` or something in the `command` block – that seems like a nice clean way to address this 🙂.

IMHO that's for a future PR though

Reply:
> IMHO that's for a future PR though

+1, I'm all for incremental improvements and small steps.